ojbackend/install.py

```diff
@@ -0,0 +1,40 @@
+"""
+	install.py
+	This file let users to set up machineInfo.config and ojdatabase.config file
+"""
+
+from bash import bash
+def set_MachineInfo():
+    
+    bash('test -d machineInfo.config  && rm -r machineInfo.config')    
```

Comment:
Using ```Popen``` can handle different messages by yourself.
On the other way, using ```check_call``` will notify the user when procedure call fails.
But I'm not sure whether ```bash``` can get the similar results.